CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -0,0 +1,30 @@
+Note that we start to track changes starting from v1.3.7.
+
+<hr>
+
+## v1.3.7(TBD)
+
+### BoltDB
+- Add [recursive checker to confirm database consistency](https://github.com/etcd-io/bbolt/pull/225).
+- Add [support to get the page size from the second meta page if the first one is invalid](https://github.com/etcd-io/bbolt/pull/294).
+- Add [support for loong64 arch](https://github.com/etcd-io/bbolt/pull/303).
+- Add [internal iterator to Bucket that goes over buckets](https://github.com/etcd-io/bbolt/pull/356).
+- Add [validation on page read and write](https://github.com/etcd-io/bbolt/pull/358).
+- Add [PreLoadFreelist option to support loading free pages in readonly mode](https://github.com/etcd-io/bbolt/pull/381).
+- Add [(*Tx) CheckWithOption to support generating human-readable diagnostic messages](https://github.com/etcd-io/bbolt/pull/395).
+- Fix [Use `golang.org/x/sys/windows` for `FileLockEx`/`UnlockFileEx`](https://github.com/etcd-io/bbolt/pull/283).
+- Fix [readonly file mapping on windows](https://github.com/etcd-io/bbolt/pull/307).
+- Fix [the "Last" method might return no data due to not skipping the empty pages](https://github.com/etcd-io/bbolt/pull/341).
```

Comment:
I would stress this 2 items and potentially improving correctness/stability. 

```
##Summary

Release v1.3.7 contains following critical fixes:
  - fix to problem that `Last` method might return incorrect value (#341)
  - fix of potential panic when performing transaction's rollback (#362)

Other changes focused on defense-in-depth (#358, #294, #225, #395)
 
`bbolt` command line tool was expanded to:
  - allow fixing simple corruptions by `bbolt surgery` (#370)
  - be flexible about output formatting (#306, #359)
  - allow accessing data in subbuckets (#295)
```

Reply:
How about get these info included in the release TEXT? such as in https://github.com/etcd-io/bbolt/releases/tag/v1.3.5

Reply:
After second thought, Let me update the changelog

Reply:
thx & updated.